HelgaShiryaeva/assignment1/Anagram2.java

```diff
@@ -0,0 +1,87 @@
+package assignment1;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.ArrayList;
+import java.util.StringTokenizer;
+
+public class Anagram {
```

Comment:
No need to keep two files. Only one class with final version should be left in PR for merging.